### PR TITLE
Comments and suggestiosn on whole manuscript

### PR DESCRIPTION
**Pull request by jfy133**
* [ ] fix #(issue number)
* [ ] description of feature/fix
* [ ] tests added/passed
* [ ] add an entry to the [changelog](../CHANGELOG.md)

### CODE REVIEW

**Comment by jfy133** [comment]
Some further comments below, I still need to make suggestions on the statement of need

My read-through notes:

- [ ] Affiliations: Doesn't Sofia have 3?
- [x] Summary: isn't necessarily correct: can be eukaruotes (not micrbes),
- [x] Summary: first sentence is history that doesn't necessarily prime the main question
- [x] Summary: Missing wrap up sentence
- [x] SoN: Wrong citation MEGAN/MALT
- [x] SoN: maybe shortly describe example problem of varying profiler output formats
- [x] SoN: Condense other tools sentences into one pargraph, mention kraken2tools(?), QIIME2 is mostly aimed at 16S etc, doesn't CAMI challenge also have a sort of format (more generic)?
- [ ] SoN: Missing caveats (which it does/doesn't do, e.g. doesn't do analysis, fixed output to read counts)
- [x] Acknolwegments: Max Planck Society
- [x] Acknowedgments: RAPID-Seq should have quotes 

Questions:

- [ ] TAXPASTA or taxpasta or Taxpasta? Should pick one and use everywhere (e.g. docs has TAXPASTA, taxpasta, Taxpasta)
- [ ] Do we care/want a simple image summarising the tool, like a graphical summary/figure?

Shower thoughts:

- [x] should explicitly mention normally researchers would have to write their own parsers for each and every of the tools
- [x] even biom format would require data parting to get it into the right format, AFAICR it didn't have parses for different tools
- [x] Krakentools only works for a specific 'family' of profilers (kraken and derivatives)

File: paper/paper.md
Comment (Midnighter) [inline paper/paper.md:1]: I'm adding your changes with a manual merge and will modify parts afterwards.

**Review by Midnighter — COMMENTED** [review]
I'm closing this PR and merging the changes manually. I think, it's a bit step forward and your mental notes make a lot of sense. I will try to refine it a bit more. I think, we also need some more references for what we're trying to say in the introduction.

File: paper/paper.md
Comment (jfy133) [reply paper/paper.md:1]: Sounds good!

File: paper/.gitignore
[the remark this answers is not in the document]
Comment (jfy133) [reply paper/.gitignore:1]: wasn't me :grimacing:

**Pull request closed without merging**